app/assets/stylesheets/components/common/_news_signup_sticky.scss

```diff
@@ -0,0 +1,131 @@
+// Newsletter signup sticky footer component
+//
+// Styleguide Newsletter Signup Sticky
+
+.news-signup-sticky {
+  @extend %stripe-bg;
+  @include clearfix;
+  background-color: $color-off-white;
+  margin: $baseline-unit*2 0 0 0;
+  text-align: center;
+  position: relative;
+}
+
+.news-signup-sticky--positioned {
+  visibility: hidden;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  z-index: 100;
```

Comment:
I notice a lot of z-index: [0-9] going on in the frontend codebase - worth a seperate PR to tidy up maybe?